### PR TITLE
feat(gux-context-menu): adopt new tokens

### PR DESCRIPTION
**Pull request by thomas-c-dillon**
COMUI-2290

### CODE REVIEW

File: packages/genesys-spark-components/src/components/stable/gux-context-menu/gux-context-menu.scss [hunk 1]
```diff
@@ -7,26 +7,20 @@
...
 }
 
 button.gux-compact {
-  height: var(
-    --gse-ui-contextMenu-button-compact
-  ); // TODO: [COMUI-2290] name should be more specific
-
-  padding: 0 4px; // TODO: [COMUI-2290] missing token
+  height: var(--gse-ui-button-compact-height);
+  padding: 0 4px;
```
Comment (thomas-c-dillon) [inline packages/genesys-spark-components/src/components/stable/gux-context-menu/gux-context-menu.scss:23]: Open question in the ticket comments if tokens should be used for the padding.

Comment (daragh-king-genesys) [reply packages/genesys-spark-components/src/components/stable/gux-context-menu/gux-context-menu.scss:23]: I think they should be.

Comment (MattCheely) [reply packages/genesys-spark-components/src/components/stable/gux-context-menu/gux-context-menu.scss:23]: +1, we shouldn't have any hardcoded `px` values.

**Review by MattCheely — CHANGES_REQUESTED** [review]

File: packages/genesys-spark-components/src/components/stable/gux-context-menu/gux-context-menu.scss (hunk 1 above)
Comment (thomas-c-dillon) [reply packages/genesys-spark-components/src/components/stable/gux-context-menu/gux-context-menu.scss]: New tokens have been added!

**Review by jordanstith15 — APPROVED** [review]

**Comment by thomas-c-dillon** [comment]
@MattCheely I think this PR is blocked from merging until you approve the PR because you previously requested changes.

**Review by MattCheely — APPROVED** [review]

**Pull request merged**